### PR TITLE
Add release notes for 1.5.3

### PR DESCRIPTION
**Pull request by kgilmer**

### CODE REVIEW

**Comment by kgilmer** [comment]
Assuming this is good, I'll merge it when the package sync has completed from `stable` to `release`.  I haven't started that yet.

File: content/en/docs/Reference/Releases/regolith-1.5.3-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "1.5.3 Release Notes"
+linkTitle: "1.5.3 Release Notes"
+weight: 1
+description: >
+  Release notes for Regolith 1.5.3.
```
Comment (moritzheiber) [inline content/en/docs/Reference/Releases/regolith-1.5.3-release-notes.md]: Smallest of nitpicks: can you remove the dot at the end of the line? The description gets translated into a h3 or h4, which looks weird ending in a dot

**Comment by moritzheiber** [comment]
LGTM, with the small, optional!, change

File: content/en/docs/Reference/Releases/regolith-1.5.3-release-notes.md (hunk 1 above)
Comment (kgilmer) [reply content/en/docs/Reference/Releases/regolith-1.5.3-release-notes.md]: Yep, fixed thanks :)

**Pull request merged**